File: БР1.1/Захматов Юрий/labs/lab1/src/main/java/com/petproject/itmoacsbackend/booking/dto/BookingResponse.java
package com.petproject.itmoacsbackend.booking.dto;

import com.petproject.itmoacsbackend.booking.enums.BookingStatus;
import com.petproject.itmoacsbackend.payments.dto.PaymentResponse;
import lombok.Builder;

import java.time.LocalDate;
import java.time.LocalDateTime;

@Builder
public record BookingResponse(
        Long id,
        Long renterId,
        Long propertyId,
        PaymentResponse payment,
        LocalDate startDate,
        LocalDate endDate,
        BookingStatus status,
        Double totalPrice,
        Integer guestsCount,
        String details,
        LocalDateTime createdAt
) {
}

File: БР1.2/Цатинян Артём/labs/lab1/src/main/kotlin/ru/itmo/restaurantbooking/booking/adapter/rest/dto/BookingResponse.kt
package ru.itmo.restaurantbooking.booking.adapter.rest.dto

import java.time.LocalDateTime

data class BookingResponse(
    val id: Long,
    val status: String,
    val guestsCount: Int,
    val startsAt: LocalDateTime,
    val endsAt: LocalDateTime,
    val specialRequests: String?,
    val createdAt: LocalDateTime,
    val restaurant: BookingRestaurantResponse,
    val table: BookingTableResponse
)
